unicode-data.cabal

```diff
@@ -17,6 +17,8 @@ build-type:          Simple
 tested-with:         GHC==8.2.2
                    , GHC==8.6.5
                    , GHC==8.8.3
+                   , GHC==8.8.4
```

Comment:
Why do we have both 8.8.3 and 8.8.4? Instead of 8.8.3 we can have a CI for 8.4.x.